coverage-check.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
dont add other stuff to your repository for CI.  Especially do not add it in the top level of your repository.

In .travis.yml, "install" section, fetch the file using curl/wget.

And place the script into a [cached directory](https://docs.travis-ci.com/user/caching/), so you can avoid fetching if it exists.